### PR TITLE
Jetpack: Update changelog for 12.2

### PR DESCRIPTION
**Pull request by anomiex**
## Proposed changes:

* Squash all the alpha version changelogs into the beta.
* Proofread and trim.
* Update link in to-test.md.

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Testing instructions:

* Proofread. See if anything I trimmed should come back, or if anything I left should be moved to a different section r removed.

### CODE REVIEW

File: projects/plugins/jetpack/CHANGELOG.md [hunk 1]
```diff
@@ -3,256 +3,102 @@
...
+- Image CDN: Replace Photon implementation with the image-cdn package. [#30050]
+- Maps: Add Mapkit maps. [#29098]
+- Modules list: Update to current styles. Visual refresh, if you will. [#30282]
+- My Plan: Display the correct plan for Jetpack Security and Backup 2-year plans. [#31013]
+- My Plan: Hide the "My Plan" on My Plan page if user has a paid project and a free plan. This way, users won't have much or conflicting information on that page. [#30804]
+- My Plan: Properly display plans with no expiration date. [#30519]
+- Newsletters: Add misconfiguration warning functionality and improve design. [#30446]
+- Newsletters: Add newsletter access level to the post edit page. [#30172]
+- Newsletters: Add Post Publish Panel. [#30668]
+- Newsletters: Enable on self-hosred environments. [#30952]
```
Comment (samiff) [inline projects/plugins/jetpack/CHANGELOG.md]:
```suggestion
- Newsletters: Enable on self-hosted environments. [#30952]
```

File: projects/plugins/jetpack/CHANGELOG.md [hunk 2]
```diff
@@ -3,256 +3,102 @@
...
+- Contact Form: Improve spam filtering. [#30755]
+- Dashboard: Add Jetpack Boost module. [#30877]
+- Forms: Improve styling of response emails. [#30088]
+- Forms: Introduce Multiple Choice and Single Choice style variations. [#30319]
+- Forms: Update pattern modal default view to Grid. [#28906]
+- Image CDN: Replace Photon implementation with the image-cdn package. [#30050]
+- Maps: Add Mapkit maps. [#29098]
+- Modules list: Update to current styles. Visual refresh, if you will. [#30282]
+- My Plan: Display the correct plan for Jetpack Security and Backup 2-year plans. [#31013]
+- My Plan: Hide the "My Plan" on My Plan page if user has a paid project and a free plan. This way, users won't have much or conflicting information on that page. [#30804]
```
Comment (samiff) [inline projects/plugins/jetpack/CHANGELOG.md]:
```suggestion
- My Plan: Hide the "My Plan" on My Plan page if user has a paid product and a free plan. [#30804]
```

Comment (anomiex) [reply projects/plugins/jetpack/CHANGELOG.md]: Huh, I thought I cleaned that one up already.

**Review by samiff — DISMISSED** [review]

**Review by samiff — APPROVED** [review]

**Comment by anomiex** [comment]
Cherry-picked to the release branch in 7566b83f63f4d0a93e9376231f74c883044e8c1b

**Pull request merged**